.travis.yml

```diff
@@ -1,9 +1,14 @@
 language: node_js
 node_js:
-  - "6"
+- '6'
 before_script:
-  - npm install
-  - echo '{}' > config/local.json
+- npm install
+- echo '{}' > config/local.json
 script: npm test
 notifications:
   slack: siliconvalleybootcamp:SnRpv9p0fK5FmSEFa7iVqJAw
+deploy:
+  app: goober
+  provider: heroku
+  api_key:
+    secure: I0UNPhpj2iX8QZeyfRp+PrazFFa7Jfbg4e2Nz/LvgY2JLdpjqoglXBG5JjBD303ZBueEA+odjFsdOFqAmUDiOb1VShGmBZ/tp+Msq7AnqWxv8siS+bZcVyRi7SV6cPR9DHxUi3e4Y/XFovh1zY0Mc7npe1H6/QP0Z+UUzzWmQrRWPhWr6F4Z6xEPvFzoQN9yzbs/YBp+JeL5ZD5y62eghzP4MgwcbaBLcqVN9Ux6uhlEd3OouK3b/08Os1ApBgkVsa1/BcDjATl5RHfagPeK7M3OZ1nUt7ahiCRpitIEytY5eF08uqtSwk1GE0jnfFOe6CUpNtamUet29914/3GPWeUi3zd+zwuqpazKXAJUmCpSBhLZju5+9IcwTd4YUHzbWs5jo4Sa8Z/GzeyH1PiDXqprqhhYWRVD5NtXMpZa1XkKQBZFLb6woZU+B5PInPa5ndMphcGzhWX399+9nB7tGAX967L+Y2F9/0sbjhtT+/+OUriemj8x36LU1q5a20wSJW4scaLzyzTIfTr8uDoeX2N75VE0yw12fHLg/8xnnLkWzVMYYXNBE3V4Yhz8CMCyUL/abDnH31wIQNhI/WliJeR98nL+mGXBhuE59cSTi5D6o6TMdGKXBrGTwyzpGmnToNezg6eKVKYA/917PG5t+O+VKZhyxsnAcACV4NQ6ZLo=
```

Comment:
Do not include this information in the repo.  Please discard this key immediately.